tests/handlers/aws/test_outputs.py

```diff
@@ -0,0 +1,190 @@
+
+import gzip
+from share import json_dumper, json_parser
+import boto3
+import os, os.path
+
+from main_aws import handler
+
+from typing import Any
+
+import mock
+import pytest
+from unittest import TestCase
+from testcontainers.localstack import LocalStackContainer
+from tests.testcontainers.logstash import LogstashContainer
+from string import Template
+# main test facilities
+from tests.handlers.aws.test_handler import ContextMock
+# test helpers
+from tests.handlers.aws.test_handler import _s3_event_to_sqs_message, _event_from_sqs_message
+import datetime
+import botocore
+
+def load_file_fixture(name:str) -> str:
+    filepath = os.path.join(os.path.dirname(__file__), 'testdata', name)
+
+    res = ""
+    with open(filepath) as f:
+        res = f.read()
+
+    return res
+
+def _upload_content_to_bucket(client, content, key:str, bucket_name:str, content_type:str, acl:str="public-read-write") -> None:
```

Comment:
I see this is still a draft: in order to target main or the feature branch I would aim for having a `test_integration_utils.py` with this and all the other helpers below without duplication of what we already have around with similar functionality.

indeed this could be a pre-emptive separate PR: let's separate all the helpers in `test_integration_utils.py`, replace the current usage, and rely on new helpers from then onwards.

such PR can go directly to main

Reply:
Indeed! My goal here was to understand how the code was working. One key difference between this version and the one in the other file is that this function takes `client` as an argument, instead of relying on `_mockaws_client`. This avoids client-reuse issues (as you control which client is being used).

Reply:
I extracted the functions in a separate file in [2b031f6](https://github.com/elastic/elastic-serverless-forwarder/pull/186/commits/2b031f68d3523c800947201068a6c4ef793748b2)